modules/nw-dpu-creating-a-sfc.adoc

```diff
@@ -18,27 +18,55 @@ Follow this procedure on the DPU to create the network function `my-network-func
 
 .Procedure
 
-. Save the following YAML file example as `sfc.yaml`:
+. Save the following YAML file example as `dpu-pod.yaml`. This is an example of a simple pod that will be scheduled directly onto a DPU node.
```

Comment:
What is doing the scheduling here? We shouldn't be using passive tense.

Reply:
k8s default scheduler